usefulUtilities/statusIndicator/hifiScript/zoneEntityScripts/locationZoneClient.js

```diff
@@ -8,8 +8,8 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 
 (function() {
-    var request = Script.require('request').request,
-        REQUEST_URL = "http://localhost:3305/",
+    var request = Script.require(Script.resolvePath('./modules/request.js')).request,
```

Comment:
please use the released version of the request module instead of adding another copy to the repo

Reply:
got it